src/Microsoft.Azure.SignalR/HubHost/ServiceConnection.cs

```diff
@@ -102,11 +102,18 @@ private async Task ProcessOutgoingMessagesAsync(ServiceConnectionContext connect
             }
             catch (Exception ex)
             {
+                // The exception means applicaion fail to process input anymore
+                // Turn down application output to avoid new messages and notify client to disconnect
                 Log.SendLoopStopped(_logger, connection.ConnectionId, ex);
+                connection.Application.Output.CancelPendingFlush();
```

Comment:
is it enough to set `connection.AbortOnClose`?

Reply:
AbortOnClose is default true. Do you mean no need to WriteAync(CloseMessage) since WaitOnApplicationTask will do this?